packages/orchestration/src/exos/chain-hub.js

```diff
@@ -336,6 +357,33 @@ export const makeChainHub = (agoricNames, vowTools) => {
       // @ts-expect-error XXX generic parameter propagation
       return lookupChainsAndConnection(primaryName, counterName);
     },
+
+    /**
+     * Register an asset that may be held on a chain other than the issuing
+     * chain.
+     *
+     * @param {import('../orchestration-api.js').Denom} denom - on the holding
+     *   chain, whose name is given in `detail.chainName`
+     * @param {DenomDetail} detail - chainName and baseName must be registered
+     */
+    registerAsset(denom, detail) {
+      const { chainName, baseName } = detail;
+      chainName === 'agoric' ||
```

Comment:
Confused by this special case. Doesn't `chainInfos` have 'agoric'?

Reply:
ok, I suppose it should